apps/codelab/src/app/codelabs/angular/create-first-app/output.json

```diff
@@ -0,0 +1,43 @@
+{
+  "s":[
```

Comment:
Love having questions in JSON! 
What's "s" property though?
can you get rid of unnecessary nesting?